securedrop/manage.py

```diff
@@ -167,11 +169,45 @@ def add_admin():
             break
 
     while True:
+        print "Please enter a password.\n\nThe password must be at least 8 characters and contain a combination of upper\nand lower characters, numbers and special characters.\n"
         password = getpass("Password: ")
         password_again = getpass("Confirm Password: ")
         if password == password_again:
-            break
-        print "Passwords didn't match!"
+            print "Your password is " + str(len(password)) + " characters long."
+            strength = ['blank','very weak','weak','medium strength','medium-strong','strong', 'very strong']
```

Comment:
You don't need so many possible evaluations of the password strength. "Acceptable" and "not acceptable" would be enough :smile: